docs/source/_autogen/user-apis.rst

```diff
@@ -1,5 +1,5 @@
-API Reference
-=============
+User API
```

Comment:
Honestly this tab is still impossible to read since it's one giant page. Definitely need to find a way to break this up into smaller sections.

Reply:
In my work I got something nice working. Still one page, but it had nested sections for each module and package. Will port that over at some point.